### ISSUE TITLE
Bootstrap Issue while setting up the project

### ISSUE THREAD

**Issue by KomalAgarwal1** [linnovate/openideal#209]
Hi Team,

I am trying to setup the openideal project via dockerised solution (using link https://github.com/linnovate/openideal/wiki/Install-OpenideaL-with-Docker4Drupal-by-Wodby)

While performing 5th step in php container I am getting below error. Can you please help on resolving the same?

Execute this command in folder web/ to install drupal with the Openideal profile drush si -y --account-name admin --account-pass 123 --account-mail my_mail@example.com --site-name "Openidea L" --db-url=mysql://drupal:drupal@mariadb/openideal idea

========================

> Using PHP 7.4.29
> PHP is using the following php.ini file:
  /usr/local/etc/php/php.ini

> Checking Box requirements:
  ✔ The package "composer/xdebug-handler" requires the version "^5.3.2 || ^7.0 || ^8.0" or greater.
  ✔ The package "laravel-zero/phar-updater" requires the version "^7.3|^8.0" or greater.
  ✔ The package "psr/log" requires the version ">=5.3.0" or greater.
  ✔ The package "symfony/polyfill-ctype" requires the version ">=7.1" or greater.
  ✔ The package "webmozart/assert" requires the version "^7.2 || ^8.0" or greater.
  ✔ The package "webmozart/path-util" requires the version ">=5.3.3" or greater.
  ✔ The package "webflo/drupal-finder" requires the extension "json".

 [OK] Your system is ready to run the application.

 [preflight] Config paths: /var/www/html/vendor/drush/drush/drush.yml
 [preflight] Alias paths: /var/www/html/drush/sites
 [preflight] Commandfile search paths: /var/www/html/vendor/drush/drush/src
 [debug] Starting bootstrap to root [12.31 sec, 8.72 MB]

In BootstrapHook.php line 32:

  [Exception]
  Bootstrap failed. Run your command with -vvv for more information.  

Exception trace:
  at /var/www/html/vendor/drush/drush/src/Boot/BootstrapHook.php:32
 Drush\Boot\BootstrapHook->initialize() at /var/www/html/vendor/consolidation/annotated-command/src/Hooks/Dispatchers/InitializeHookDispatcher.php:34
 Consolidation\AnnotatedCommand\Hooks\Dispatchers\InitializeHookDispatcher->callInitializeHook() at /var/www/html/vendor/consolidation/annotated-command/src/Hooks/Dispatchers/InitializeHookDispatcher.php:27
 Consolidation\AnnotatedCommand\Hooks\Dispatchers\InitializeHookDispatcher->initialize() at /var/www/html/vendor/consolidation/annotated-command/src/CommandProcessor.php:145
 Consolidation\AnnotatedCommand\CommandProcessor->initializeHook() at /var/www/html/vendor/consolidation/annotated-command/src/AnnotatedCommand.php:289
 Consolidation\AnnotatedCommand\AnnotatedCommand->initialize() at /var/www/html/vendor/symfony/console/Command/Command.php:221
 Symfony\Component\Console\Command\Command->run() at /var/www/html/vendor/symfony/console/Application.php:1005
 Symfony\Component\Console\Application->doRunCommand() at /var/www/html/vendor/symfony/console/Application.php:255
 Symfony\Component\Console\Application->doRun() at /var/www/html/vendor/symfony/console/Application.php:148
 Symfony\Component\Console\Application->run() at /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php:118
 Drush\Runtime\Runtime->doRun() at /var/www/html/vendor/drush/drush/src/Runtime/Runtime.php:48
 Drush\Runtime\Runtime->run() at /var/www/html/vendor/drush/drush/drush.php:72
 require() at /var/www/html/vendor/drush/drush/includes/preflight.inc:18
 drush_main() at phar:///usr/local/bin/drush/bin/drush.php:143
 require() at /usr/local/bin/drush:14

**Comment by KomalAgarwal1** [comment]
Can anyone please help on this ?

**Comment by heyyo-droid** [comment]
Did you execute the drush command inside the container ?
You should not have such error.

**Comment by KomalAgarwal1** [comment]
@heyyo-droid  Yes I have executed the command inside the container inside web folder
